libnavui-speedlimit/src/main/java/com/mapbox/navigation/ui/speedlimit/model/SpeedData.kt

```diff
@@ -0,0 +1,18 @@
+package com.mapbox.navigation.ui.speedlimit.model
+
+import com.mapbox.navigation.base.formatter.UnitType
+import com.mapbox.navigation.base.speed.model.SpeedLimitUnit
+
+/**
+ * Defines the [speed] that needs to be formatted using [PostedAndCurrentSpeedFormatter], [fromUnit]
+ * [toUnit]
+ *
+ * @param speed can either be posted or current speed
+ * @param fromUnit should be meters/sec for current speed and kilometers/hour for posted speed
+ * @param toUnit can either be [UnitType.METRIC] or [UnitType.IMPERIAL]
+ */
+data class SpeedData internal constructor(
```

Comment:
It probably shouldn't be a `data class`.

Reply:
Done.